packages/vow/vat.js

```diff
@@ -26,8 +26,7 @@ export const defaultPowers = harden({
 export const prepareVowTools = (zone, powers = {}) =>
   rawPrepareVowTools(zone, { ...defaultPowers, ...powers });
 
-export const vowTools = prepareVowTools(makeHeapZone());
-export const { watch, when, makeVowKit, allVows } = vowTools;
+export const heapVowTools = prepareVowTools(makeHeapZone());
```

Comment:
Alternately, if this whole module is heap stuff, we could keep the existing exports and rename the module `heap-vows.js`.

@michaelfig either way, can you suggest a `/** @file` comment ?

Reply:
The whole module is not heap stuff, it is SwingSet stuff.  You're probably lookin' for sumpin' like:

```js
/**
 * @file specialization of the `@agoric/vow` package for the vat disconnection rejections produced by
 * the SwingSet kernel.
 */

/** Produce SwingSet-compatible vowTools, with an arbitrary Zone type */
export const prepareVowTools = ...;

/** `vowTools` that are not durable, but are useful in non-durable clients that need to consume vows from
 * other SwingSet vats. */
export const heapVowTools = ...;
```